### PR TITLE
Bug 1441835 Added pods_per_core to scaling and performance guide

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1441835

Built view: http://file.bne.redhat.com/bfallonf/19_04_2017/podspercore_1441835/scaling_performance/host_practices.html#scaling-performance-capacity-host-practices-node

cc @sferich888 @jeremyeder

### CODE REVIEW

**Comment by sferich888** [comment]
LGTM

**Comment by bfallonf** [comment]
Thanks @sferich888 .

@ahardin-rh @adellape @mburke5678 peer review, please? I moved around some of the info and included a section from another part of the guides using the include/tags thing.

File: admin_guide/manage_nodes.adoc [hunk 1]
```diff
@@ -430,14 +430,32 @@ openshift_node_kubelet_args={'max-pods': ['40'], 'resolv-conf': ['/etc/resolv.co
...
+* Slow pod scheduling.
+* Potential out-of-memory scenarios (depends on the amount of memory in the node).
+* Exhausting the pool of IP addresses.
+* Resource overcommitting, leading to poor user application performance.
+
+[NOTE]
+====
+In Kubernetes, a pod that is holding a single container actually uses two
+containers. The second container is used to set up networking prior to the
+actual container starting. Therefore, a system running 10 pods will actually
```
Comment (mburke5678) [inline admin_guide/manage_nodes.adoc]: Does this second container remain after the actual container starts?

**Comment by mburke5678** [comment]
LGTM with one question.

**Comment by ahardin-rh** [comment]
LGTM

**Comment by jeremyeder** [comment]
ack, thank you @bfallonf !

File: admin_guide/manage_nodes.adoc (hunk 1 above)
Comment (bfallonf) [inline admin_guide/manage_nodes.adoc]: I presume so? That's what I understand, and what I understand it to be saying. This is from the whitepaper, so unless it's changed, I'll presume this is okay.

**Comment by bfallonf** [comment]
Seems this is all fine. I'll go ahead and merge it. Thanks for the eyes, all.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../scaling_performance/host_practices.adoc#scaling-performance-capacity-host-practices[Recommended Host Practices]
|Included a section on setting max pods per node.
%

**Pull request merged**